### ISSUE TITLE
ASN get network

### ISSUE THREAD

**Issue by seiimonn** [maxmind/GeoIP2-python#76]
As far as I can see it is only possible to get the IP back in the model for most query types.
I would be interested to get the whole range back in which the IP was found.

As example if I would query the ip address "1.0.0.12" to the ASN database I would expect an answer with following fields:

- autonomous_system_number -> 13335
- autonomous_system_organization -> Cloudflare, Inc.
- network ->  1.0.0.0/24

Is something like this archivable?

**Comment by seiimonn** [comment]
I do not think this is a duplicate. In my use case I would just like to know which entry in the database matched. Somehow it is decided to which entry in the database a given ip belongs to. Therefor I assume all information should already be available.

**Comment by oschwald** [comment]
As mentioned in the other story, this data isn't currently exposed by the underlying `maxminddb` reader. We plan on exposing that information there in the future and at that time we will likely add a network` property to the returned object in `geoip2`.

**Comment by smartip-io** [comment]
Is there any update on this matter? I think this would be a very useful info to be surfaced.

**Comment by oschwald** [comment]
This is available in master.
